en/mkdocs.yml

```diff
@@ -825,9 +825,7 @@ nav:
               - Callout Mediator: reference/mediators/callout-mediator.md
               - Class Mediator: reference/mediators/class-mediator.md
               - Clone Mediator: reference/mediators/clone-mediator.md
-              - Data Mapping:
-                  - Data Mapper Mediator: reference/mediators/data-mapper-mediator.md
-                  - Data Mapper Schema Specification: reference/mediators/data-mapper-json-schema-specification.md
```

Comment:
Since we are removing this page, shall we add a redirection too, to avoid 404 when switching between versions?